### DISCUSSION TITLE
can't find terminal with id

### DISCUSSION THREAD

**Discussion by Yahud100** [eclipse-theia/theia#14346] [Q&A]
2024-10-10T17:11:32.988Z terminal WARN Couldn't attach - can't find terminal with id: 1502366150 
while running in debug mode application is getting stopped and throwing this error on debug console , but which running application using yarn start getting error in console but application is working fine , unable to debug because of this error , can anyone help me on this ?

**Comment by msujew** [comment]
Hey @Yahud100,

this is a known issue, see https://github.com/eclipse-theia/theia/issues/12733 and https://github.com/eclipse-theia/theia/issues/13609. Note that you can close the terminal before starting to debug the app and everything should work as expected. This only seems to affect Windows at the moment.

**Reply by Yahud100** [reply]
thanks for your response , yes as you mentioned i have checked if any terminal is opened in theia , there is no terminal opened , still i am getting this terminal problem while starting the application in debug mode , please help me to sort this

**Reply by msujew** [reply]
There is no fix for this right now. It's a very deep rooted issue in our dependency that manages the terminal input/output stream.

**Reply by Yahud100** [reply]
thank you
